bigbluebutton-html5/imports/ui/components/video-dock/video-menu/container.jsx

```diff
@@ -25,13 +45,33 @@ export default withTracker((params) => {
   const user = Users.findOne({ userId: Auth.userID });
   const userLocked = mapUser(user).isLocked;
 
+  const isConecting = (!isSharingVideo && isConnected);
+  const isLocked = (LockCam && userLocked) || webcamOnlyModerator;
+  const isDisabled = isLocked
+      || isWaitingResponse
+      || isConecting
+      || enableShare;
+
+  const baseName = Meteor.settings.public.app.basename;
+  const videoItems = [
```

Comment:
These objects shouldn't be created in the `withTracker`.
Pass down the dynamic data that Meteor has to track and then construct these objects further down the road.

Reply:
I kinda disagree with this point. By having those itens upper on the scope it will be much easier to integrate when we actually add the functionality and multiple layout types.

Reply:
"Easier to integrate" by defining onClick handlers, labels and descriptions in the Meteor Reactive `withTracker`? How does it help exactly?

Reply:
The list will depend upon on another service when the other layout comes. Moving it to the Container component instead of the tracker does the job.

Reply:
Sure, that's what I'm talking about, moving it to the container.

Reply:
Oh, I thought you wanted them to be moved to the `JoinVideoOptions` bellow the container. :+1: